README.md

```diff
@@ -0,0 +1,75 @@
+# laravel-weather-apis
+
+## Run the app via Docker
+
+Build a Docker image for the app:
+
+Please ask Erika for the weather api key.
+```bash
+docker build --tag weather-app --build-arg token=<YOUR_AUTH_TOKEN> --build-arg key=<WEATHER_API_KEY> .
+```
+
+Run the Docker container with the built image above
+
+```bash
+docker run -d -p 8000:8000 weather-app
+```
+
+And try the curl commands below!
+
+---
+
+## If you're interested in setting up PHP...
+
+Need to install the following:
+
+- [PHP](https://www.php.net/manual/en/install.macosx.packages.php)
+- [Composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-macos)
+- [gRPC for PHP](https://cloud.google.com/php/grpc)
+
+Once the above are installed, run:
+
+```bash
+composer install
+```
+
+You need to the following env variables:
+
+- `API_KEY` this is for weather API. Ask Erika for the key.
```

Comment:
This should be changed to provide instructions for users to generate their own API key.